src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Network/AddAzureVirtualIP.cs

```diff
@@ -24,6 +24,14 @@ namespace Microsoft.WindowsAzure.Commands.ServiceManagement.IaaS.Network
     [Cmdlet(VerbsCommon.Add, "AzureVirtualIP"), OutputType(typeof(ManagementOperationContext))]
     public class AddAzureVirtualIP : ServiceManagementBaseCmdlet
     {
+        public AddAzureVirtualIP()
```

Comment:
Added a parameterized constructor, so default constructor is required to support backward compatibility.